kratos/mpi/tests/test_mpi_processes.py

```diff
@@ -24,9 +24,6 @@ def testComputeNodalGradientProcess(self):
         input_filename = GetFilePath("auxiliar_files_for_python_unittest/mdpa_files/test_mpi_serializer")
         ReadModelPart(input_filename, main_model_part)
 
-        ParallelFillCommunicator = KratosMultiphysics.mpi.ParallelFillCommunicator(main_model_part)
-        ParallelFillCommunicator.Execute()

```

Comment:
Just to be sure, you're removing these because you're always using the parallel importer right?

Reply:
Yes, this could just be in a clean up PR instead, but since Im editing the file anyway I kept here.

I saw the paralell fill is already being called in the importer so these lines can be removed